src/pages/en/guides/data-fetching.md

```diff
@@ -4,35 +4,109 @@ title: Data Fetching
 description: Learn how to fetch remote data with Astro using the fetch API.
 ---
 
-Astro components and pages can fetch remote data to help generate your pages. Astro provides two different tools to pages to help you do this: **fetch()** and **top-level await.**
+Astro pages can fetch remote data at build time to help generate your pages.
 
-## `fetch()`
+## `fetch()` in Astro
 
-Astro pages have access to the global `fetch()` function in their setup script. `fetch()` is a native JavaScript API ([MDN<span class="sr-only">- fetch</span>](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch)) that lets you make HTTP requests for things like APIs and resources.
+All [Astro components](/en/core-concepts/astro-components) have access to the global `fetch()` function in their component script to make HTTP requests to APIs. This fetch call will be executed at build time, and the data will be available to the component template for generating dynamic HTML. 
```

Comment:
What do you think about using “`.astro` files” as a hand-wavy way to talk about both components and pages?

```suggestion
All `.astro` files have access to the global `fetch()` function in their component script to make HTTP requests to APIs. This fetch call will be executed at build time, and the data will be available to the component template for generating dynamic HTML. 
```

Reply:
I'm thinking maybe I leave this one here (because I like having an excuse to link to Astro components), and changing the one above into `.astro` files instead of pages and components. (Because pages are just a special case of components anyway, and saying pages and components could inadvertantly give the impression of excluding layouts, eg.)